ci/run-k8s-e2e-tests.sh

```diff
@@ -138,6 +140,11 @@ case $key in
     SYSTEMD_LOGS_IMAGE="$2"
     shift 2
     ;;
+    --rancher-testbed)
+    RANCHER=true
+    DEFAULT_E2E_CONFORMANCE_SKIP="${DEFAULT_E2E_CONFORMANCE_SKIP}|\[sig-network\]"
```

Comment:
May I know the background info of this PR? Why we skip sig-network in rancher test?

Reply:
sig-network test is not supported in the current testing version of rancher

Reply:
Could you be more specific about "not supported"? Which tests fail? Due to test itself, rancher, or antrea issue? There should an issue with details first to justify the PR and to follow up.

Reply:
Sure, i will create an issue for the same.

Reply:
its a rancher testbed related issue for sig-network[network-policy] test

Reply:
@jainpulkit22 I saw you mentioned `[sig-network] Ingress API should support creating Ingress API operations [Conformance]` was not supported for the Rancher cluster. If so, we can only skip this test case, not the entire sig-network test suite.

Reply:
1. Skipping the entire test suite just because one case doesn't make sense.
2. There is no detail or reference why the test failed so it can't justify why it should be skipped.
3. Even if the test should be skipped, the current way complicates the script unnecessarily, it should just parameterize the skipped test when invoking the script, instead of customiing the script for rancher.

Reply:
Using `[sig-network]` will cause all tests with this label to be skipped. You can refer to our configuration in the `NETWORKPOLICY_SKIP` to skip individual test case. https://github.com/antrea-io/antrea/blob/683a72f37adbe1da6008cf520880b9c2ee41ea75/ci/jenkins/test.sh#L50